pawnjester/More-recipesApp
# #154611541 - implement-view-count

#### What does this PR do?
* allows users to view the number of views a recipe has

#### Description of Task to be completed?
- edit server side code to include view count
- implement it on the server side

#### How should this be manually tested?
Have the app working by:
 * run the command `npm runs start:dev` on the command line to start the server
 * once app is running, navigate to the app using `localhost:3000/recipes`
* click on a recipe to see the number of views

 #### Any background context you want to provide?
 Nil
 #### What are the relevant pivotal tracker stories?
#154611541 - implement-view-count

 #### Screenshots (if appropriate)
<img width="1386" alt="screen shot 2018-01-25 at 10 49 30 am" src="https://user-images.githubusercontent.com/26750279/35381711-7576caaa-01bd-11e8-85f9-6417d018e166.png">
